src/controllers/user.js

```diff
@@ -0,0 +1,37 @@
+import {trimUser} from '../util/user';
+
+/**
+ * User Controller Initialization Function
+ * @param {Object} ControllerParams - ControllerParams
+ * @param {Object} ControllerParams.userModel - Sequelize User Model
+ * @returns {Object} Controller Object
+ */
+export default ({userModel}) => {
+    const addToWatchlist = async (req, res, next) => {
+        try {
+            const user = await userModel.findOne({where: {email: req.user.email}});
+
+            if(!user){
+                const userNotFoundError = new Error(`User with email - ${req.user.email} Not Found`);
+                userNotFoundError.statusCode = 404;
+                throw userNotFoundError;
+            }
+
+            const updatedUser = await user.update({
+                watchlist: [...user.dataValues.watchlist, req.body.movieId] 
```

Comment:
Are you verifying for the correct movie ID?

Reply:
Now that you mention it, I am not right now
I didn't think there was a way to verify it but an idea just hit me on  how I can do that 

I'll add that now

Reply:
All right